### DISCUSSION TITLE
State tree enhancements

### DISCUSSION THREAD

**Discussion by klauswiesel** [reduxjs/redux-devtools#1027] [Ideas]
Hi, we are working on an angular app and store is growing. So I would appreciate, if there would be an option to sort the state tree in alphabetical order, at least in the root level. A filter in the tree would also be awesome...

**Comment by Methuselah96** [comment]
Thanks for the suggestion. Track and up-vote https://github.com/reduxjs/redux-devtools/issues/433 to see when this gets implemented.
